src/calculateRank.js

```diff
@@ -1,3 +1,21 @@
+// https://stackoverflow.com/a/5263759/10629172
+function normalcdf(mean, sigma, to) {
```

Comment:
I really have no idea what this does but i'm just experimenting it's a Cumulative distribution function : https://stackoverflow.com/questions/5259421/cumulative-distribution-function-in-javascript/5263759#5263759